### PR TITLE
Add a Wiki document on how to get started and contribute to Elixir template

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/nimblehq/elixir-templates/issues/160

## What happened

Add a Wiki document on how to get started and contribute to Elixir template.

## Insight

- Add `Contribution.md`

## Proof Of Work

https://github.com/nimblehq/elixir-templates/pull/175/files

### CODE REVIEW

File: .github/wiki/Contribution.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Adding a new addon
+
+There are 3 types of addons
+1. API
+2. Live
+3. Web
+
+For example, to add a new web addon, create a new file at `lib/nimble_template/addons/variants/phoenix/web/sample_addon.ex`
+
+```
```
Comment (longnd) [inline .github/wiki/Contribution.md:10]: we can imply the language for code highlighting :) 
```suggestion
```elixir
```

Comment (topnimble) [reply .github/wiki/Contribution.md]: Fixed in 10c2bea

File: .github/wiki/Contribution.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
+# Adding a new addon
+
+There are 3 types of addons.
+1. API
+2. Live
+3. Web
```
Comment (andyduong1920) [inline .github/wiki/Contribution.md:6]: we have Mix addon as well Top :) 
image

**Review by andyduong1920 — APPROVED** [review]
Thanks @topnimble I just have a small comment :D

File: .github/wiki/Contribution.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
```
Comment (andyduong1920) [inline .github/wiki/Contribution.md:7]: We can also explain the addons structure by adding this image, what do you think @topnimble 
Pasted_Image_08_03_2022__17_57

Comment (topnimble) [reply .github/wiki/Contribution.md:7]: Fixed in bf6fea9

File: .github/wiki/Contribution.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
+# Adding a new addon
+
+There are 2 types of addon, Mix addon and Phoenix addon.
+
+Phoenix addon consists of 3 types.
+1. API
+2. Live
+3. Web
+
+[[https://user-images.githubusercontent.com/11751745/157225788-9f8c28ad-058b-4994-851e-82c98854ddcd.png|alt=structure]]
```
Comment (andyduong1920) [inline .github/wiki/Contribution.md:10]: Hi Top, I think we need to upload this file to the Wiki folder 😄 

image

And then we can refer that image by `diagram`

Comment (topnimble) [reply .github/wiki/Contribution.md]: Fixed in ab169c9

File: .github/wiki/Contribution.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
+# Adding a new addon
+
+There are 2 types of addon, Mix addon and Phoenix addon.
+
+Phoenix addon consists of 3 types.
+1. API
+2. Live
+3. Web
+
+![diagram](assets/images/diagram.png)
```
Comment (andyduong1920) [inline .github/wiki/Contribution.md:10]: is it `diagram` 🤔 should we rename to `addon-structure`, what do you think @topnimble

File: .github/wiki/Contribution.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+Generator.print_log("* applying ", inspect(__MODULE__))
+```
+
+# Run unit test on local
+
+`mix test`
+
+# Apply the whole template on local
+
+See [[Installation]]
```
Comment (andyduong1920) [inline .github/wiki/Contribution.md:240]: Note: We will change this link `[[Installation]]` once we merged the other PR #171

File: .github/wiki/_Sidebar.md [hunk 7]
```diff
@@ -4,5 +4,6 @@
 
 ## Architecture
 
+- [[Contribution]]
```
Comment (andyduong1920) [inline .github/wiki/_Sidebar.md:7]: should we move this part out of `Architecture` module @topnimble

File: .github/wiki/Contribution.md (hunk 5 above)
Comment (topnimble) [reply .github/wiki/Contribution.md]: Fixed in a4835fe

File: .github/wiki/Contribution.md (hunk 6 above)
Comment (topnimble) [reply .github/wiki/Contribution.md]: Fixed in c50e06b

**Review by andyduong1920 — APPROVED** [review]

File: .github/wiki/Contribution.md [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+Generator.print_log("* applying ", inspect(__MODULE__))
+```
+
+# Run unit test on local
+
+`mix test`
+
+# Apply the whole template on local
+
+See [[Test The Apply Action On Development]]
```
Comment (longnd) [inline .github/wiki/Contribution.md:240]: where does this link to? I don't think we have a WIKi file with that name.

Comment (longnd) [reply .github/wiki/Contribution.md:240]: never mind. I found that in another PR #171

**Review by longnd — APPROVED** [review]

**Pull request merged**